### DISCUSSION TITLE
TS100 tip not detected

### DISCUSSION THREAD

**Discussion by GirafiStudios** [Ralim/IronOS#1638] [Q&A]
So my TS100 suddenly stopped recognizing both my iron tips. According to the information I can found about it, it's most likely the heater coil in the TS100 that have died.

The [HardwareIssues.md](https://github.com/Ralim/IronOS/blob/dev/Documentation/HardwareIssues.md) says the following about how to double check if that is actually the case
> The best way to see if this is the case is to measure the resistance across the contacts to the tip using a multimeter. you are expecting to see measurements in the range of 4-10 ohms. Anything higher than 10 ohms is generally an issue.

Anybody that knows what "the contacts" being referenced here, is exactly? 

Any other tips on how to troubleshoot this would be helpful as well!

**Comment by discip** [comment]
What is meant are the contacts at the tip itself.

Additionally you could try the follwing: https://github.com/Ralim/IronOS/issues/1601#issuecomment-1473125616

**Reply by GirafiStudios** [reply]
Thanks for the quick reply!
Gotcha, so I would need to check the the Ohm reading between the + & - as shown on the picture. The readings I get for my two tips is ~7,8 & ~9,8
I just took apart my TS100 again & noticed that the "copper ring" (As shown on the picture below, was bent in the top. I bent it back & now it works fine again!

![TS100 Mini Soldering Iron Programable_9-1000x750](https://user-images.githubusercontent.com/6305377/228369990-85b74962-b271-4d4e-86cb-355efbf50757.jpg)

**Reply by discip** [reply]
Glad it's working again! 😊

**Reply by River-Mochi** [reply]
> Thanks for the quick reply! Gotcha, so I would need to check the the Ohm reading between the + & - as shown on the picture. The readings I get for my two tips is ~7,8 & ~9,8 I just took apart my TS100 again & noticed that the "copper ring" (As shown on the picture below, was bent in the top. I bent it back & now it works fine again!
> 

Those tips should be between 7.8 ohms - 8.3 ohms.   9.8 seems  a bit over spec.  try wiping cartridge  contacts on white end with IPA and testing to see if you get closer to 8.0 ohms. 

fwiw, that copper ring is the ground ring. if you test it there should be continuity between that ring and the  rear GND screw mount. 
that round ring comes in contact with the silver of the cartridge and touches the silver barrel metal tunnel hole. important reference it needs to be installed first before the  PCB is placed into the handle.

**Reply by GirafiStudios** [reply]
Thanks for the advice, I'll try that next time I need to solder something :)

I figured as much with the copper ring, I just hadn't noticed that it was bent when I took it apart initially :)

**Comment by River-Mochi** [comment]
@GirafiStudios  as the issue is fixed now by bending the copper ring back, could you hit the purple button to close the ticket?

**Reply by GirafiStudios** [reply]
Looks like it already got closed, by the time I saw your message :)
